utils/mvn/utils.go

```diff
@@ -113,6 +120,11 @@ func runMvnVersionCommand(mavenHome string) ([]string, error) {
 }
 
 func parseMvnHome(line string) (string, error) {
+	// Line example: 'Maven home: /usr/share/maven'
```

Comment:
I think it would be cleaner to put this comment above the function with a short explanation, as Godoc.